src/cljs/rems/util.cljs

```diff
@@ -75,9 +86,14 @@
   Additionally calls event hooks."
   [url opts]
   (js/window.rems.hooks.get url (clj->js opts))
-  (GET url (merge {:response-format :transit}
-                  opts
-                  {:error-handler (wrap-default-error-handler (:error-handler opts))})))
+  ;; TODO: change also put! and post! to return a promise?
+  (p/create
+   (fn [resolve reject]
+     (GET url (-> (merge {:response-format :transit}
+                         opts
+                         {:error-handler (wrap-default-error-handler (:error-handler opts))})
+                  (update :handler append-handler resolve)
+                  (update :error-handler append-handler reject))))))
```

Comment:
please document that fetch returns a promise (but that you can ignore the return value if you don't need it)